composer.json

```
@@ -68,12 +73,32 @@
 	"autoload-dev": {
 		"psr-4": { "Imagify\\Tests\\": "Tests/"	}
 	},
+	"extra": {
+		"mozart": {
+			"dep_namespace": "ImagifyPlugin\\Dependencies\\",
+			"dep_directory": "/src/Dependencies/",
```

```suggestion
			"dep_directory": "/classes/Dependencies/",
```
Our namespaced files are in the `classes` directory, so we can use that for autoloading the dependencies too